src/main/java/org/springframework/data/couchbase/repository/support/DynamicInvocationHandler.java

```diff
@@ -44,25 +44,37 @@
 	final ReactiveCouchbaseTemplate reactiveTemplate;
 	CommonOptions<?> options;
 	String collection;
-	String scope;;
+	String scope;
 
 	public DynamicInvocationHandler(T target, CommonOptions<?> options, String collection, String scope) {
 		this.target = target;
 		if (target instanceof CouchbaseRepository) {
 			reactiveTemplate = ((CouchbaseTemplate) ((CouchbaseRepository) target).getOperations()).reactive();
 			this.entityInformation = ((CouchbaseRepository<?, String>) target).getEntityInformation();
 		} else if (target instanceof ReactiveCouchbaseRepository) {
-			reactiveTemplate = (ReactiveCouchbaseTemplate) ((ReactiveCouchbaseRepository) target).getOperations();
-			this.entityInformation = ((ReactiveCouchbaseRepository<?, String>) target).getEntityInformation();
+			reactiveTemplate = (ReactiveCouchbaseTemplate) ((ReactiveCouchbaseRepository) this.target).getOperations();
+			this.entityInformation = ((ReactiveCouchbaseRepository<?, String>) this.target).getEntityInformation();
 		} else {
-			throw new RuntimeException("Unknown target type: " + target.getClass());
+			printInterfaces(target.getClass(), "  ");
+			throw new RuntimeException("Unknown target type: " + target.getClass()
+					+ " CouchbaseRepository.class.isAssignable:" + CouchbaseRepository.class.isAssignableFrom(target.getClass()));
 		}
 		this.options = options;
 		this.collection = collection;
 		this.scope = scope;
 		this.repositoryClass = target.getClass();
 	}
 
+	void printInterfaces(Class clazz, String tab) {
```

Comment:
is this a leftover (with the print above) from development? if log, then to the logger or into  a string?